x-pack/agentbeat/dev-tools/packaging/packages.yml

```diff
@@ -55,6 +60,14 @@ shared:
       <<: *binary_files
       <<: *windows_osquery_files
 
+  - &fips_binary_spec
```

Comment:
```suggestion
  - &binary_spec_fips
```

Reply:
done in c6e70d9fdfa2992199520f2fcffa39c757b3c8ad

Reply:
mh, I see it is done in the [‎dev-tools/packaging/packages.yml](https://github.com/elastic/beats/commit/c6e70d9fdfa2992199520f2fcffa39c757b3c8ad#diff-9e8bcd1359f1e06bbdf2c89db711c6c400ed4142cf9764ff03e6e963013122e5) but not in this file. Is this intentional?

Reply:
Simply forgot that agentbeat has its own package spec YAML :sweat_smile: It's fixed now